cluster/vm-atomic.yaml

```diff
@@ -0,0 +1,28 @@
+metadata:
+  name: atomic-ephemeral
+apiVersion: kubevirt.io/v1alpha1
+kind: VM
+spec:
+  domain:
+    devices:
+      disks:
+      - type: ContainerRegistryDisk:v1alpha
+        source:
+          name: kubevirt/fedora-atomic-registry-disk-demo:devel
+        target:
+          dev: vda
+      interfaces:
+      - source:
+          network: default
+        type: network
+    memory:
+      unit: MB
+      value: 1024
+    os:
+      type:
+        os: hvm
+    type: qemu
+  cloudInit:
```

Comment:
yep yep, my cloud-init functional tests actually use cirros.   I need to check to see if cirros actually uses the cloud-init method i'm using though. It's possible i'm just attaching a file disk to cirros with  userdata that cirros ignores.